rc/base/tmux.kak

```diff
@@ -22,7 +22,7 @@ define-command -hidden -params 1.. tmux-new-impl %{
         tmux_args="$1"
         shift
         if [ $# -ne 0 ]; then kakoune_params="-e '$@'"; fi
-        TMUX=$tmux tmux $tmux_args "env TMPDIR='${TMPDIR}' kak -c ${kak_session} ${kakoune_params}" < /dev/null > /dev/null 2>&1 &
+        TMUX=$tmux tmux $tmux_args "env TMPDIR='${TMPDIR}' kak -c ${kak_session} ${kakoune_params}" < /dev/null > /dev/null &
```

Comment:
This will prevent the command from running detached

Reply:
Actually the whole chain of redirection is uncessary, it will work without any of them as well.

Reply:
No it won't, I've tested those changes and they work? I can close the first client, and the second one will remain usable.

Have you tested the PR?

Reply:
I dont think we understood each other, the tmux command is going to return relatively quickly, its just sending a command to the tmux server. Yet there is a difference between the previous behaviour, where that tmux command was fired, then forgotten and the shell returned immediately, and the new behaviour would be to wait for that command to return (which, again, is not going to wait for the pane to be closed, just for the command to be acknowledged by the tmux server).

It would be ok not to launch that tmux command in parallel, I merely wanted to point out that this behaviour was changing, as it did not look that was obvious to you (as you did not remove the whole redirection and backgrounding).